apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -207,6 +310,11 @@ function TokenDetails({ coinType }: TokenDetailsProps) {
 
 	const tokenBalance = BigInt(coinBalance?.totalBalance ?? 0);
 
+	const { rawValue: rawBalanceInUSD } = useSuiBalanceInUSDC(
```

Comment:
This being here vs in a component that is itself wrapped in `isDefiWalletEnabled` means we hit deepbook always, we should probably gate this so that we actually gate the deepbook API calls themselves as well as the presentation of the UI

Reply:
Updated to have the hook called inside `WalletBalanceUSD` component